data/io.github.zefr0x.hashes.metainfo.xml.in

```diff
@@ -19,16 +19,6 @@
     <launchable type="desktop-id">io.github.zefr0x.hashes.desktop</launchable>
     <content_rating type="oars-1.0"></content_rating>
     <translation type="gettext" source_locale="en_GB">io.github.zefr0x.hashes</translation>
-    <categories>
-        <category>Development</category>
-    </categories>
-    <keywords>
-        <keyword>hash</keyword>
-        <keyword>hashcat</keyword>
-        <keyword>john</keyword>
-        <keyword>detect</keyword>
-        <keyword>identify</keyword>
-    </keywords>
```

Comment:
I knew that these values will be parsed from the `.desktop` file, but I don't remember why I kept them anyway.

I can't remember the argument, but I will delay this tell I do or never do ;)

Reply:
"If there’s a [launchable](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#launchable) defined for a desktop application, they are pulled from the desktop file and merged in the [AppStream Catalog data](https://www.freedesktop.org/software/appstream/docs/chap-CatalogData.html). So defining them separately in the MetaInfo is not strictly necessary." 

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#categories-and-keywords